index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div id="main">
+        <section id="navigation">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a href="index.html">Youtube</a></li>
+                </ul>
+            </nav>
+        </section>
```

Comment:
- [ ] Kindly make sure you tell the student to remove this `section` tag here as it's not supposed to be there. instead, they should use the `header` tag
You can check [here](https://www.w3schools.com/tags/tag_section.asp) when to use `section` and when to use `header` [here](https://www.w3schools.com/tags/tag_header.asp)